src/main/java/com/google/sps/servlets/UserSettingsServlet.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/settings")
+public class UserSettingsServlet extends HttpServlet {
+  // The only reason this class exists rn is to save this regular expression I made.
+  // It checks that user names are alphanumeric and (5 - 25) characters.
```

Comment:
I understand this, but I think you should but this in another classes comments or as a function in a util? Having an entire servlet to hold a comment doesn't seem like the right approach.